src/styles/_themes.scss

```diff
@@ -58,6 +58,31 @@
   --form-input-bg: transparent;
   --form-input-text: #{$theme-black};
   --form-input-border: #{color(base-light)};
+
+  // News
+  --btn-box-bg: #{$theme-cosmiclatte-lighter};
+  --btn-box-border: #{$neutral-gray};
+  --date-icon-bg: #{$theme-cosmiclatte-lighter};
+
+  // Tags
+  --source-tag-bg: #{$label-source-lighttheme-bg};
+  --source-tag-text: #{$theme-mars-lightest};
+  --source-tag-svg: #{$label-source-lighttheme-icon};
+  // TODO: couldn't find an example of these in Figma, so verify with design
+  --audience-tag-bg: #{$label-audience-lighttheme-bg};
+  --audience-tag-text: #{$theme-mars-lightest};
+  --audience-tag-svg: #{$label-audience-lighttheme-icon};
+  --base-tag-bg: #{$label-base-lighttheme-bg};
+  --base-tag-text: #{$white};
+  --base-tag-svg: #{$label-base-lighttheme-icon};
+
+  // Categories
+  // TODO: couldn't find an example of these in Figma, so verify with design.
+  // Also, does external news == public USSF news? May need to change the name of these variables.
```

Comment:
yeah i believe external news is public ussf, like what we pull into the RSS feed.